yelyzaveta.kozhevnikova/task2.rb

```diff
@@ -0,0 +1,3 @@
+h1 = {'Yes' => 23, 'b' => 'travel', 'yesterday' => 34, 5 => '234', yesss: :fg, try: 30, key: 'some value', ' yesterday1 '=> 34,' yesteryear '=> 2014}
+h2 = h1.select {|k| k.to_s.downcase.match(/^yes/)}
```

Comment:
```suggestion
h2 = h1.select { |k| k.to_s.downcase.match(/^yes/) }
```

https://github.com/rubocop-hq/ruby-style-guide#spaces-operators
It's better to add spaces so every part of your code is stand out visually